pages/mentor.page.js

```diff
@@ -124,10 +119,10 @@ function Mentor() {
 
             {/* card3 */}
             <div className="tw-group tw-flex-wrap tw-flex-1 tw-flex tw-justify-center tw-items-center tw-gap-6 tw-flex-col tw-p-12 tw-bg-gray-200 hover:tw-transform hover:tw-ease-in-out tw-duration-200 tw-rounded-md hover:tw-scale-110 tw-text-center">
-              <AutoStoriesIcon
-                fontSize=""
-                className="group-hover:tw-text-[#00C9A7] group-hover:bg-primary-200 group-hover:tw-animate-bounce  tw-text-white tw-text-9xl tw-bg-primary-100 tw-p-8 tw-rounded-full"
-              />
+
+              
```

Comment:
@Nishitbaria  and @anmode , do you really think that this is a good implementation for this SvgMaker component, I am not sure about this. 
I think we rather than giving svgPath we pass react icon with font size width and height props. 
What are both of your views?

Reply:
Hey there, @Isha988 ! 😊 I am using SvgMaker Component for creating reusable SVG Components 🎨✨ and also passing too many props with SvgPath will look too much, which means it can get a bit cluttered and reduce readability. 🧹👀